diff -Nur gnome-applets-3.5.92-old/gweather/gweather-pref.c gnome-applets-3.5.92/gweather/gweather-pref.c
--- gnome-applets-3.5.92-old/gweather/gweather-pref.c	2013-05-23 22:39:41.708158708 +0200
+++ gnome-applets-3.5.92/gweather/gweather-pref.c	2013-05-23 22:40:38.184202931 +0200
@@ -29,7 +29,6 @@
 
 #define GWEATHER_I_KNOW_THIS_IS_UNSTABLE
 
-#include <libgweather/gweather-xml.h>
 #include "gweather.h"
 #include "gweather-pref.h"
 #include "gweather-applet.h"
