--- gnome-applets-3.8.1/configure.ac.orig	2014-10-18 18:35:29.621539481 +0100
+++ gnome-applets-3.8.1/configure.ac	2014-10-18 18:35:41.068538638 +0100
@@ -100,8 +100,8 @@
 dnl *** Use pkg-config to check for dependancies                            ***
 dnl ***************************************************************************
 
-dnl -- Check for gnome-icon-theme (required) ----------------------------------
-PKG_CHECK_MODULES(GIT, gnome-icon-theme >= $GNOME_ICON_THEME_REQUIRED)
+dnl -- Check for adwaita-icon-theme (required) ----------------------------------
+PKG_CHECK_MODULES(GIT, adwaita-icon-theme >= $GNOME_ICON_THEME_REQUIRED)
 
 dnl -- check for gio (required) ------------------------------------------
 PKG_CHECK_MODULES(GIO, gio-2.0 >= $GIO_REQUIRED)
